### PR TITLE
Add hint about a potentially required configuration reset of containerd

### PR DESCRIPTION
**Pull request by ramobis**
Hi all

I'm new to Kubernetes and currently working on an education project in computer science class at university. During the work on a project with Kubernetes I had the problem that the control plane containers ended in a crash loop after the deployment of the cluster. Finally I could resolve the problem by resetting the containerd configuration file to default. Probably the provided configuration of the containerd.io package maintained by docker contains some incompatible configuration options.

The added paragraph in the info-box would have saved me hours of troubleshooting. So I decided to suggest the addition, hoping to save other people some time.

Regards

### CODE REVIEW

File: content/en/docs/setup/production-environment/container-runtimes.md [hunk 1]
```diff
@@ -217,6 +217,8 @@ that the CRI integration plugin is disabled by default.
 You need CRI support enabled to use containerd with Kubernetes. Make sure that `cri`
 is not included in the`disabled_plugins` list within `/etc/containerd/config.toml`;
 if you made changes to that file, also restart `containerd`.
+
+If you experience container crash loops after the initial cluster installation or after installing a CNI, the containerd configuration provided with the package might contain incompatible configuration parameters. Consider resetting the containerd configuration with <code>containerd config default > /etc/containerd/config.toml</code> as specified in [getting-started.md](https://github.com/containerd/containerd/blob/main/docs/getting-started.md#advanced-topics) and then set the configuration parameters specified above accordingly.
```
Comment (sftim) [inline content/en/docs/setup/production-environment/container-runtimes.md:221]: @ramobis, please wrap this text to match the rest of the Markdown in the source file.

**Review by sftim — COMMENTED** [review]
Thank you for the suggestion. We'd like some initial tweaks to make this change easier to review.

File: content/en/docs/setup/production-environment/container-runtimes.md (hunk 1 above)
Comment (ramobis) [reply content/en/docs/setup/production-environment/container-runtimes.md]: @sftim, thank you for the proposal.
I applied the change accordingly.

Comment (ramobis) [reply content/en/docs/setup/production-environment/container-runtimes.md]: Somehow the checks are now failing.. But I can not view the logs. The deploy log just loads forever.

Comment (bart0sh) [reply content/en/docs/setup/production-environment/container-runtimes.md]: @ramobis can you update the PR branch to re-trigger test run?

Comment (ramobis) [reply content/en/docs/setup/production-environment/container-runtimes.md]: Hi @bart0sh
The PR branch is now up to date.

**Pull request merged**